.travis.yml

```diff
@@ -6,6 +6,10 @@ node_js:
   - "6"
 
 sudo: false
+dist: trusty
```

Comment:
Not sure about changes in `.travis.yml`, left "as is" after running `ember-cli-update`